src/axom/core/tests/core_array.hpp

```diff
@@ -1963,34 +1953,39 @@ struct HasDefault
   {
     return (member == other.member);
   }
+
+  friend std::ostream& operator<<(std::ostream& os, const HasDefault& hd)
+  {
+    return os << hd.member;
+  }
```

Comment:
This was the missing `operator<<()` overload required for `axom::Array<HasDefault>`